scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -66,11 +66,16 @@ ENV HOME /home/sof
 
 # build cross compiler
 USER sof
-RUN cd /home/sof && git clone https://github.com/thesofproject/xtensa-overlay.git && \
-	cd xtensa-overlay && git checkout sof-gcc8.1 && cd ../ && \
-	git clone https://github.com/thesofproject/crosstool-ng.git && \
+RUN cd /home/sof && \
+	git clone \
+		--depth 1 --branch sof-gcc8.1 \
+		https://github.com/thesofproject/xtensa-overlay.git && \
+	cd xtensa-overlay && cd ../ && \
```

Comment:
@crazoes I think you can get rid of this lines split by defining variables for repo names. 

1) XT_OVERLAY_REPO
2) SOF_REPO

etc.

I think the best way to do this is in steps:
1) introduce variables for repos
2) use shallow clones

Reply:
Those extra variables would create additional layers in the Docker container but I guess that is probably fine :-??